Extension/src/LanguageServer/configurations.ts

```diff
@@ -1617,7 +1617,11 @@ export class CppProperties {
         }
         const settings: CppSettings = new CppSettings(this.rootUri);
         const compilerPathAndArgs: util.CompilerPathAndArgs = util.extractCompilerPathAndArgs(!!settings.legacyCompilerArgsBehavior, resolvedCompilerPath);
+
+        // compilerPath + args in the same string isn't working yet.
+        const skipFullCommandString = !compilerPathAndArgs.compilerName && resolvedCompilerPath.indexOf(" ") >= 0;
```

Comment:
Should this be `includes(" ")`?

Reply:
Is that preferred? I was going to use it until the doc comment for it said something about converting this to a string, so I couldn't tell if there was more overhead to use that one or not.

Reply:
I believe it is preferred. The object is already a string. The doc comment just means if it's invoked a non-string it'll try to convert it to a string.

![image](https://github.com/user-attachments/assets/4999a25a-70ad-4a10-be6b-2592edf59d51)